config/jobs/kubernetes/sig-arch/kubernetes-depstat-periodical.yaml

```diff
@@ -0,0 +1,33 @@
+periodics:
+  - interval: 6h
+    name: check-dependency-stats-periodical
+    decorate: true
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    always_run: false
+    optional: true
```

Comment:
Are these two needed here? I didn't see them in [this](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-testing/coverage.yaml#L2-L23) periodical?

Reply:
you don't need these 2